src/CuratedModule.sol

```diff
@@ -3,10 +3,16 @@
 
 pragma solidity 0.8.24;
 
+import { ICuratedModule } from "./interfaces/ICuratedModule.sol";
+import { IStakingModule } from "./interfaces/IStakingModule.sol";
+
 import { CSModule } from "./CSModule.sol";
 
-contract CuratedModule is CSModule {
-    error NotImplemented();
+import { NOAddresses } from "./lib/NOAddresses.sol";
+
+contract CuratedModule is ICuratedModule, CSModule {
+    bytes32 public constant NODE_OWNER_ADMIN_ROLE =
+        keccak256("NODE_OWNER_ADMIN_ROLE");
```

Comment:
`NODE_OPERATOR_ADDRESSES_CHANGER_ROLE` ?

Reply:
I've chosen `NODE_OWNER_ADMIN` because the following reasons:
- This role essentially sets ownership of a node operator
- This role has access to a method bypassing multiple checks, hence the ADMIN suffix

Reply:
We don't have a separate `NODE` term without `OPERATOR` alongside. Consider renaming the role somehow.